library/posture.c

```diff
@@ -795,6 +797,21 @@ static bool check_running(uv_loop_t *loop, const char *path) {
         }
     }
 
+#elif __APPLE__ && __MACH__
+    int n_pids = proc_listallpids(NULL, 0);
+    unsigned long pids_sz = sizeof(pid_t) * (unsigned long)n_pids;
+    pid_t * pids = calloc(1, pids_sz);
+    proc_listallpids(pids, (int)pids_sz);
+    char proc_path[PROC_PIDPATHINFO_MAXSIZE];
+    for (int i=0; i < n_pids; i++) {
+        if (pids[i] == 0) continue;
+        proc_pidpath(pids[i], proc_path, sizeof(proc_path)); // returns strlen(proc_path)
+        if (strncasecmp(proc_path, path, sizeof(proc_path)) == 0) {
```

Comment:
I don't believe it should be case-insensitive

Reply:
Why not? the default file systems on macOS are case-insensitive:
```
$ uname -s
Darwin
$ touch Foo
$ touch foo
$ ls ?oo
Foo
```

Reply:
:exploding_head: I did not know that